src/themes/masschallenge/elements/container.overrides

```diff
@@ -0,0 +1,64 @@
+/*******************************
+         Theme Overrides
+*******************************/
+
+.container > .grid.primary {
+        display: grid;
+        grid-template-areas: "topbar" "content" "footer" "mobile-tab-nav";
+        grid-template-rows: 55px 1fr 58px;  
+    }
+    .sidebar {
+        display: none;
+    } 
+    .mobile-tab-nav {
+        display: block;
+    }
+    .grid.primary > .footer {
+        margin: 1em 0.5em 0 2em;
+    }
+    .grid.primary > .topbar{
+        grid-area: topbar;
+    }
+ 
+    .grid.primary > .content{
+        padding: @padding;
+        grid-area: content;
+        background: #fff;
+        border: 1px solid #eaeaea;
+        border-radius: 3px;
+    }
+
+@media screen and (min-width: @tabletBreakpoint){
+    .container > .grid.primary {
+        display: grid;
+        grid-column: 2;
+        grid-template-columns: 220px 960px;
+        grid-template-areas: "topbar topbar" "sidebar content" "sidebar footer";
+        grid-template-rows: 55px auto;
+        grid-gap: 2em 1em;
+    }
+    .sidebar {
+        display: block;
+    }
+    .mobile-tab-nav {
+        display: none;
+    }
+    .grid.primary > .content {
+      margin: 0 1em 2em 0;
+    }
+    .grid.primary > .sidebar{
+      grid-area: sidebar;
+      padding: @padding;
+  }
+   .menu-side{
+      position: fixed;
+      top: calc(56px + 2em);
+  }
+  .grid.primary > .footer {
+        grid-area: footer;
+        display: flex;
+        flex-wrap: wrap;
+        justify-content: space-between;
+        margin: 2.5em 0 1.5em;
+    }
+}
```

Comment:
Because you have included all of the `.footer` specification inside of a media query, there is no styling for the footer on mobile devices. I suggest refactoring this to:
**Shared `.footer` props are placed outside of media queries:**
```
.grid.primary > .footer {
  display: flex;
  flex-wrap: wrap;
  padding: 1em;
}
```
**Include only the differing properties inside of the media query:**
```
/* Mobile view */
@media screen and (max-width: @tabletBreakpoint) {
  .footer {
    flex-flow: column;
    align-items: center;
    justify-content: space-evenly;
    margin-bottom: 6em;
  }
}

/* Tablet & desktop view */
@media screen and (min-width: @tabletBreakpoint) {
  .footer {
    justify-content: space-between;
    margin: 1.5em 0em 0.5em;
  }
}
```